scanpipe/tests/test_models.py

```diff
@@ -2352,6 +2356,22 @@ def test_scanpipe_codebase_resource_queryset_elfs(self):
         self.assertTrue("e" in paths)
         self.assertTrue("a" in paths)
 
+    def test_scorecard_models(self):
+        package = DiscoveredPackage.create_from_data(self.project1, package_data1)
+        scorecard_obj = PackageScoreMixin.from_data(scorecard_data)
+        package_score = PackageScore.create_from_data(
+            package, scorecard_obj, PackageScore.ScoringTool.OSSF
+        )
+
+        self.assertIsNotNone(package_score)
+        self.assertEqual(package_score.scoring_tool, PackageScore.ScoringTool.OSSF)
+        self.assertEqual(package_score.score, "6.7")
+
+        checks = package_score.discovered_packages_score_checks.all()
+        self.assertEqual(checks.count(), 15)
+        self.assertEqual(checks[0].check_name, "Code-Review")
+        self.assertEqual(checks[0].check_score, "1")
+
 
```

Comment:
We should also check the functions which are used on the pipeline directly by mocking the data we get in the scorecard API.